dashboard/config/android/generate.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Copyright 2020 syzkaller project authors. All rights reserved.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+# https://source.android.com/setup/build/building-kernels
+
+set -eux
+
+SRC_DIR=$1
```

Comment:
We can have same `[ -z "${CC}" ]` argument checking here like in other config generation scripts.

Reply:
CC is not an argument for this script.
We use the official Android compiler, and it's defined below.

Reply:
`[ -z "${SRC_DIR}" ] ...` and `[ -z "${VERSION}" ] ...` in this case. Or change the existing scripts to make all of them consistent.

Reply:
We do the following check below:
```
case "$VERSION" in
	5.4)
	*)
```
It looks more useful than -z and makes -z redundant. Similar for SRC_DIR.

Reply:
OK